ush/config.sh.RRFS_NA_13km

```diff
@@ -33,25 +33,25 @@ QUILTING="TRUE"
 CCPP_PHYS_SUITE="FV3_RAP"
 
 EXTRN_MDL_ICS_OFFSET_HRS="3"
-BOUNDARY_LEN_HRS="36"
-BOUNDARY_LONG_LEN_HRS="36"
-LBC_SPEC_INTVL_HRS="3"
-EXTRN_MDL_LBCS_OFFSET_HRS="6"
+BOUNDARY_LEN_HRS="39"
+BOUNDARY_LONG_LEN_HRS="3"
```

Comment:
Is LONG not supposed to be a larger number than the regular value?

Reply:
No need to have long boundary process for NA 13km. This one just set a small value so it will not show as a long list.